<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHM</class>
 <widget class="QMainWindow" name="IHM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GestionBarriereParking</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>65</y>
      <width>162</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IP automate Wago</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>203</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestion d'une barrière de parking</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adresseIPWago">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>172.20.2.1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_connecter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Connecter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deconnecter">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Déconnecter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_allumerBalise">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Allumer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>145</y>
      <width>82</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Balise orange</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eteindreBalise">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Eteindre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>205</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Barrière</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_descendreBarriere">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Descendre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_monterBarriere">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Monter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quitter">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_log">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>259</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_effacerLog">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Effacer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>257</width>
      <height>196</height>
     </rect>
    </property>
    <property name="text">
     <string>image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_lireCapteurs">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Lire</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
